dist/less/order-service.less

```diff
@@ -170,17 +175,24 @@
       }
     }
   }
+  .review-message {
+    padding-left: 10px;
+  }
+  .review-status {
+    display: flex;
+    font-size: 16px;
+
+    .pficon {
+      margin-top: 3px;
+    }
+  }
 }
 
-.wizard-pf-contents {
+.order-service-wizard-step.wizard-pf-main {
   @media (min-width: @screen-sm-min) {
-    display: flex;
-    min-height: 100%;
-    padding-bottom: (@grid-gutter-width / 2);
-    padding-top: (@grid-gutter-width / 2);
-    .scroll-shadows-horizontal-covers();
-    _::-webkit-full-page-media, _:future, :root & { // only target Safari
-      background: none;
-    }
+    height: calc(@order-service-page-height + @grid-gutter-width);
```

Comment:
I'm really torn on this. Options:

1. Make the wizard resize per page

  This jumps the wizard height around even when selecting options on a single page. This is a bit jarring to me and the next/back buttons move up and down while trying to navigate quickly.

2. Keep a consistent height as it is now

This means more scrolling on pages with lots of parameters. Worse when there are a moderate amount and I have to scroll for one or two parameters when I wouldn't otherwise have to scroll at all. When there are a slew of them, I'm scrolling either way.

3. Keep a consistent height, but use as much window vertical space as possible

Less scrolling for sure, can probably fit an extra 8-10 parameters before needing to scroll. When simply selecting an existing project, the page is largely empty and lots of mouse movement to get down to the next button.